frontend_tests/node_tests/markdown.js

```diff
@@ -381,10 +403,18 @@ run_test('marked', () => {
          expected: '<p>@**O&#39;Connell**</p>'},
         {input: '@*Bobby <h1>Tables</h1>*',
          expected: '<p><span class="user-group-mention" data-user-group-id="3">@Bobby &lt;h1&gt;Tables&lt;/h1&gt;</span></p>'},
+        {input: '@*& &amp; &amp;amp;*',
+         expected: '<p><span class="user-group-mention" data-user-group-id="4">@&amp; &amp;amp; &amp;amp;amp;</span></p>'},
```

Comment:
On second thought, maybe the other behavior made more sense here from a CommonMark compatibility perspective.

Reply:
Resolved.